site/content/en/latest/design/eg-metrics.md

```diff
@@ -0,0 +1,238 @@
+---
+date: 2023-10-10
+title: "Control Plane Observability: Metrics"
+author: Xunzhuo Liu
+linkTitle: "Control Plane Observability: Metrics"
+---
+
+{{% alert title="State" color="warning" %}}
+
++ Author: [Xunzhuo Liu](https://github.com/Xunzhuo)
++ Affiliation: Tencent
++ Data: 2023-10-12
++ Status: Done
+{{% /alert %}}
+
+This document aims to cover all aspects of envoy gateway control plane metrics observability.
+
+{{% alert title="Note" color="secondary" %}}
+**Data plane** observability (while important) is outside of scope for this document.
+{{% /alert %}}
+
+## Current State
+
+At present, the Envoy Gateway control plane provides logs and controller-runtime metrics, without traces. Logs are managed through our proprietary library (`internal/logging`, a shim to `zap`) and are written to `/dev/stdout`.
+
+The absence of comprehensive and robust control plane metrics observability hinders the effective monitoring of Envoy Gateway in a production environment, a critical requirement before deploying Envoy Gateway into production.
+
+## Goals
+
+Our objectives include:
+
++ Supporting **PULL** mode for Prometheus metrics and exposing these metrics on the admin address.
++ Supporting **PUSH** mode for Prometheus metrics, thereby sending metrics to the Open Telemetry Stats sink.
```

Comment:
You don't mention HTTP or gRPC here, it seems that you use HTTP by default, so what about gRPC?